workflow/Snakefile

```diff
@@ -37,16 +37,13 @@ rule all:
     Target rule with final output of the pipeline
     """
     input:
-        PWM_input_dir = config["pwm_directory"],
-        DIR_MotEvo_results_pwm = expand(
-            os.path.join(
-                "{outdir}",
-                "motevo",
-                "{pwm_dir}"
-            ),
-            outdir = config["outdir"],
-            pwm_dir = gather_motifs_names(config["pwm_directory"])
```

Comment:
As this is the input to your rule we do not need it in the last rule of the workflow anymore.